### PR TITLE
Use lmazuel/swagger-to-sdk Docker container

### PR DESCRIPTION
**Pull request by lmazuel**
As discussed with @devigned on the Slack workflow channel, SwaggerToSdk needs its own Github repository:
- It permits to enable SwaggerToSdk.py Travis testing (rest-api travis is not the right place)
- Make a change in SwaggerToSdk.py does not regenerate the SDK (not relevant)

In addition, I created a Docker container with the configured env and the script. See the new Travis file:
- It permits to reduce the amount of code in the Travis file
- It reduces the time to migrate to Jenkins if needed one day
- It allows testing the script easily for anyone who can use Docker
- Last but not least, divide by 2x the Travis build time

https://github.com/lmazuel/swagger-to-sdk
https://hub.docker.com/r/lmazuel/swagger-to-sdk/

This commit erases the SwaggerToSdk files from this repo at the same time.

### CODE REVIEW

**Comment by lmazuel** [comment]
Build failed in Node because the Node configuration file is not using the last configuration format.

**Comment by lmazuel** [comment]
@devigned @amarzavery Is there any problem to merge this one, or it's just your are too busy on other things? :)

**Comment by devigned** [comment]
LGTM!

File: .travis.yml [hunk 1]
```diff
@@ -1,31 +1,20 @@
...
+  - python -c "import os; print('\n'.join(v for v in os.environ.keys() if v.startswith('TRAVIS')))" > /tmp/env_file
 script:
-- if [[ $MODE == 'python' ]]; then python3.5 ./SwaggerToSdk/SwaggerToSdk.py AutorestCI/azure-sdk-for-python
-  --pr-repo-id Azure/azure-sdk-for-python -o dev -v; fi
-- if [[ $MODE == 'node' ]]; then python3.5 ./SwaggerToSdk/SwaggerToSdk.py AutorestCI/azure-sdk-for-node
-  --pr-repo-id Azure/azure-sdk-for-node -o autorest -v; fi
-- if [[ $MODE == 'syntax' ]]; then npm test; fi
+  - DOCKER_CMD="docker run --rm --env-file /tmp/env_file -e GH_TOKEN -v $PWD:/git-restapi/ lmazuel/swagger-to-sdk"
+  - if [[ $MODE == 'python' ]]; then $DOCKER_CMD AutorestCI/azure-sdk-for-python --pr-repo-id Azure/azure-sdk-for-python -o dev -v; fi
+  - if [[ $MODE == 'node' ]]; then $DOCKER_CMD AutorestCI/azure-sdk-for-node --pr-repo-id Azure/azure-sdk-for-node -o autorest -v; fi
```
Comment (amarzavery) [inline .travis.yml]: -o master for node sdk

**Pull request merged**